utils/camspipeline.py

```
@@ -129,43 +149,23 @@ def create_table_if_not_exists(table_name: str, variable_name: str, engine) -> T
 
-        # Convert cftime.DatetimeGregorian to datetime.datetime
+        # Convert cftime to datetime
         if df['time'].dtype == 'object' and any(isinstance(t, cftime.DatetimeGregorian) for t in df['time']):
             df['time'] = pd.to_datetime([t.isoformat() for t in df['time']])
 
         # Verify required columns
         expected_columns = {'time', 'latitude', 'longitude', variable_name.lower()}
         if not expected_columns.issubset(df.columns):
-            raise ValueError(f"DataFrame missing required columns: {expected_columns - set(df.columns)}")
+            logger.error(f"DataFrame missing required columns: {expected_columns - set(df.columns)}")
+            raise ValueError(f"DataFrame missing required columns")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)